vector/src/main/java/im/vector/app/features/location/preview/LocationPreviewFragment.kt

```diff
@@ -81,9 +79,7 @@ class LocationPreviewFragment :
         views.mapView.onCreate(savedInstanceState)
 
         viewLifecycleOwner.lifecycleScope.launch {
-            viewLifecycleOwner.repeatOnLifecycle(Lifecycle.State.CREATED) {
```

Comment:
I don't know if this code was here for a certain reason, but I agree that it does not seem necessary since we're in the `onViewCreated` method